rules/README.md

```diff
@@ -1,6 +1,6 @@
 # rules/
 
-Rules within this folder are organized by solution or platform. The structure is flattened out, because nested file hierarchies are hard to navigate and find what you're looking for. Each directory contains several [.toml](https://github.com/toml-lang/toml) files, and the primary ATT&CK tactic is included in the file name when it's relevant (i.e. [`windows/execution_via_compiled_html_file.toml`](windows/execution_via_compiled_html_file.toml))  
+Rules within this folder are organized by solution or platform. The structure is flattened out, because nested file hierarchies are hard to navigate and find what you're looking for. Each directory contains several [.toml](https://github.com/toml-lang/toml) files, and the primary ATT&CK tactic is included in the file name when it's relevant (i.e. [`windows/execution_via_compiled_html_file.toml`](windows/execution_via_compiled_html_file.toml))
```

Comment:
what changed on this line?

Reply:
trailing space was deleted. I'm happy to put it back

Reply:
during difficulties of the line ending stuff, I had done a blanket "put back" of extra trailing newlines, and the editor deleted the trailing spaces on each line.

Reply:
ah no need to put it back. i was looking at this earlier on my phone and its highlighting isn't smart enough to show whitespace